lib/pipely/build/template.rb

```diff
@@ -12,6 +12,8 @@ module Build
     class Template
       include TemplateHelpers
 
+      attr_accessor :pipeline_id
```

Comment:
Why not make :pipeline_id an attr_accessor directly on Definition?  I don't see how it is ever used directly on Template